includes/Modules/Reader_Revenue_Manager/Tag_Interface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Reader_Revenue_Manager\Tag_Interface
+ *
+ * @package   Google\Site_Kit\Modules\Reader_Revenue_Manager
+ * @copyright 2025 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Reader_Revenue_Manager;
+
+/**
+ * Interface for a Reader_Revenue_Manager tag.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+interface Tag_Interface {
```

Comment:
This is a bit confusingly named, because the RRM `Web_Tag` already ultimately implements an interface called `Tag_Interface` via `Tag`:

https://github.com/google/site-kit-wp/blob/8842999952733331794fdcdbdd44222a365131f7/includes/Core/Tags/Tag.php#L22

I'd suggest renaming it to something more specific e.g. `Reader_Revenue_Manager_Tag_Interface` or `RRM_Tag_Interface`.

Reply:
Hmm - on second thoughts, do we even need this interface? It's only used in `Web_Tag`, we're not passing anything around as a `Tag_Interface`... I think we can simply remove this interface altogether.

Reply:
This was added following the Analytics `Tag_Interface` class, but I agree that it doesn't do much, and I've removed it as suggested. Thanks!